css/editorialBio.css

```diff
@@ -0,0 +1,12 @@
+.orcidContent{
+    display:flex;
+}
+
+.orcidIcon{
+    width:24px; 
+    height:24px;
+}
+
+.orcidUrl{
+    font-size:24px;
```

Comment:
Fonts generally should not be bound to specific pixel measurements.  This inhibits dynamic resizing, for example in the case of individuals with low vision.  Rather, text size is based on `em`s.  Here, I'm unconvinced of the need to increase the text size, but if we did, I think we would want it more subtle, like 1.2em.